src/components/podcast/Header.js

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+
+/**
+ * Component for displaying the header of a podcast
+ * @param {*} props podcast object
```

Comment:
Love seeing js doc tags for components. Very nice. It's also good practice to use PropTypes for all your components to help other people (including future-you) know when they're passing an invalid prop for some reason or another.